mautic/mautic
# Fix bad date format.

Signed-off-by: Giovanni Mascellani <giovanni@mascellani.eu>

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Issue(s) addressed                     | Trivial typo, no need for an issue




#### Description:



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Notice how log message now have the correct date instead of showing the month in the place of the minutes
